packages/core/redux/bitcoin/constants.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+export const DUCK_BITCOIN = 'bitcoin'
+
+export const TX_UPDATE = 'tx/update'
```

Comment:
Let's rename it to BITCOIN_TX_UPDATE and change its value to 'BITCOIN/TX/UPDATE'